### ISSUE TITLE
How to debug pixel history of render target?

### ISSUE THREAD

**Issue by waltdestler** [MicrosoftDocs/visualstudio-docs#6922]
I'm trying to figure out how to debug the pixel history of a render target that is *not* my final frame buffer but can find no mention of how to do this in the docs. It would be great if this was added.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: aaff607d-8e67-eeee-bad6-811cec20b1b0
* Version Independent ID: 5ba73f6b-c289-53d6-cbe8-b6199c35499b
* Content: [Walkthrough: Debugging Rendering Errors Due to Shading - Visual Studio (Windows)](https://docs.microsoft.com/en-us/visualstudio/debugger/graphics/walkthrough-debugging-rendering-errors-due-to-shading?view=vs-2019)
* Content Source: [docs/debugger/graphics/walkthrough-debugging-rendering-errors-due-to-shading.md](https://github.com/MicrosoftDocs/visualstudio-docs/blob/master/docs/debugger/graphics/walkthrough-debugging-rendering-errors-due-to-shading.md)
* Product: **visual-studio-windows**
* Technology: **vs-ide-debug**
* GitHub Login: @Mikejo5000
* Microsoft Alias: **mikejo**

**Comment by Mikejo5000** [comment]
Hi, thanks for the feedback. Unfortunately, we aren't currently planning additional doc updates in this area, because Pix on Windows is the [recommended replacement tool](https://docs.microsoft.com/en-us/visualstudio/debugger/graphics/visual-studio-graphics-diagnostics?view=vs-2019).

**Comment by waltdestler** [comment]
> Hi, thanks for the feedback. Unfortunately, we aren't currently planning additional doc updates in this area, because Pix on Windows is the [recommended replacement tool](https://docs.microsoft.com/en-us/visualstudio/debugger/graphics/visual-studio-graphics-diagnostics?view=vs-2019).

Unless I'm mistaken, PIX doesn't support DX11, so it's not a replacement for my purposes.

**Comment by Mikejo5000** [comment]
Are you sure it's a doc issue and that VS supports it? I suggest you try [Report a Problem](https://docs.microsoft.com/visualstudio/ide/how-to-report-a-problem-with-visual-studio) and see if they are able to help.

**Comment by waltdestler** [comment]
Yeah I am sure, because [I actually figured out how to do it](https://stackoverflow.com/questions/68630574/how-to-view-debug-pixel-history-of-a-render-target-in-the-visual-studio-graphi).

So not really a big issue for me anymore, just figured I'd point out a way the docs could have been made more complete.

Cheers! :)

**Comment by WilliamAntonRohm** [comment]
@waltdestler -- Walt, thank you for your considerate follow-up.

@Mikejo5000 -- Mike, please look into this potential documentation issue.
